src/code.cloudfoundry.org/policy-server/store/security_groups_store.go

```diff
@@ -0,0 +1,197 @@
+package store
+
+import (
+	"fmt"
+	"strings"
+
+	"code.cloudfoundry.org/policy-server/store/helpers"
+)
+
+//counterfeiter:generate -o fakes/security_groups_store.go --fake-name SecurityGroupsStore . SecurityGroupsStore
+type SecurityGroupsStore interface {
+	Replace([]SecurityGroup) error
+	BySpaceGuids([]string, Page) ([]SecurityGroup, Pagination, error)
+}
+
+type SGStore struct {
+	Conn Database
+}
+
+func (sgs *SGStore) BySpaceGuids(spaceGuids []string, page Page) ([]SecurityGroup, Pagination, error) {
+	query := `
+		SELECT
+			id,
+			guid,
+			name,
+			rules,
+			staging_default,
+			running_default,
+			staging_spaces,
+			running_spaces
+		FROM security_groups`
+
+	whereClause := `staging_default=true OR running_default=true`
+
+	if len(spaceGuids) > 0 {
+		whereClause = fmt.Sprintf("%s OR %s OR %s",
+			whereClause,
+			sgs.jsonOverlapsSQL("staging_spaces", spaceGuids),
+			sgs.jsonOverlapsSQL("running_spaces", spaceGuids),
```

Comment:
Are there concerns with this using fmt.sprintf + embedded query values vs parameter bindings? 

The API call is providing the list of space guids - couldn't that open us up to an SQL injection attack?

Reply:
Discussed with maria - this didnt work the way i thought it did + concerns were addressed